manager/integration/tests/common.py

```diff
@@ -2433,3 +2433,24 @@ def wait_for_engine_image_deletion(client, core_api, engine_image_name):
         break
 
     assert deleted
+
+
+def create_snapshot(longhorn_api_client, volume_name):
+    volume = longhorn_api_client.by_id_volume(volume_name)
+
+    snapshots = volume.snapshotList(volume=volume_name)
+    snapshots_count = len(snapshots.data)
+
+    volume.snapshotCreate()
+
+    snapshot_created = False
+    for i in range(RETRY_COUNTS):
+        snapshots = volume.snapshotList(volume=volume_name)
+
+        if len(snapshots) == snapshots_count + 1:
+            snapshot_created = True
+            break
+        time.sleep(RETRY_INTERVAL)
+
+    assert snapshot_created
+    return snapshots
```

Comment:
Doesn't make sense for the `create_snapshot` to return a list of snapshots. Make more sense to only return the newly created snapshot's name.

Reply:
fixed